bfcfg

```diff
@@ -48,54 +48,130 @@ log_msg()
   echo "$@" >> ${log_file}
 }
 
+mfg_lock()
+{
+  log_msg "mfg: lock the partition"
+  echo 1 > /sys/bus/platform/drivers/mlx-bootctl/mfg_lock 2>>${log_file}
+}
+
 mfg_cfg()
 {
   local tmp_file=/tmp/.bfcfg-mfg-data
-  local opn_str_sysfs=/sys/bus/platform/drivers/mlx-bootctl/opn_str
+  local opn_sysfs=/sys/bus/platform/drivers/mlx-bootctl/opn
```

Comment:
It seems cleaner to add a variable like below at line 38~42
mfg_sysfs_path=/sys/bus/platform/drivers/mlx-bootctl